physical/postgresql/postgresql.go

```diff
@@ -63,6 +114,12 @@ func NewPostgreSQLBackend(conf map[string]string, logger log.Logger) (physical.B
 		maxParInt = physical.DefaultParallelOperations
 	}
 
+	var hae bool = false
+	haestr, ok := conf["ha_enabled"]
```

Comment:
`ok` isn't needed since the zero value is "" and the check below works fine without.

Reply:
Addressed in PR from @ncabatoff

Reply:
fixed in PR by ncabatoff